setup.py

```diff
@@ -119,7 +108,7 @@
       cmdclass=cmdclassd,
       zip_safe=False,
       entry_points=entry_points,
-      python_requires='>=' + __minimum_python_version__,
+      python_requires='>=3.5',
```

Comment:
Nit picking, but I wonder there is any sense in getting this from one config place? e.g. `ah_bootstrap.py` also has it, etc.

Reply:
It'll be in ``setup.cfg`` after https://github.com/astropy/astropy/pull/8240 - will rebase that PR after/if this is merged.